src/sagemaker/pytorch/estimator.py

```diff
@@ -23,7 +23,12 @@
     python_deprecation_warning,
     is_version_equal_or_higher,
 )
-from sagemaker.pytorch.defaults import PYTORCH_VERSION, PYTHON_VERSION, LATEST_VERSION
+from sagemaker.pytorch.defaults import (
+    PYTORCH_VERSION,
+    PYTHON_VERSION,
+    LATEST_VERSION,
+    LATEST_PY2_VERSION,
+)
```

Comment:
at this point it might be worth starting to do `from sagemaker.pytorch import defaults` instead 😂